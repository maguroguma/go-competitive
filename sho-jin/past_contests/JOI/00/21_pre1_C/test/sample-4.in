10 10
76 91 98 7 98 7 98 10 63 91
70 71 10 10 91 70 65 10 63 76
