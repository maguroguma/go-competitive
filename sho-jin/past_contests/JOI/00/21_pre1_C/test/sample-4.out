10
63
76
91
